app/controllers/api/v2/discovered_hosts_controller.rb

```diff
@@ -62,6 +63,7 @@ def create
         param :medium_id, String, :desc => N_("required if not imaged based provisioning and host is managed and value is not inherited from host group")
         param :ptable_id, :number, :desc => N_("required if host is managed and custom partition has not been defined")
         param :subnet_id, :number, :desc => N_("required if host is managed and value is not inherited from host group")
+        param :subnet6_id, :number, :desc => N_("required if host is managed, does not have IPv4 IP / Subnet, or the value is not inherited from the host group")
```

Comment:
What does it mean ` does not have IPv4 IP / Subnet`?

Reply:
The host must pass one of these conditions:

* Have IPv4 address or IPv4 subnet
* Have IPv6 address or IPv6 subnet
* Have IPv4 & IPv6 addresses
* Have IPv4 & IPv6 subnets

Calling for @Lennonka for text review / suggestions.

Reply:
```suggestion
        param :subnet6_id, :number, :desc => N_("required if host is managed, has neither IPv4 IP nor IPv4 subnet, or the value is not inherited from the host group")
```
Hopefully this is clear :)